<ui version="4.0" >
 <class>preferencesDialog</class>
 <widget class="QDialog" name="preferencesDialog" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Preferences</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons.qrc" >
    <normaloff>:/icons/papywizard.png</normaloff>:/icons/papywizard.png</iconset>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>-1</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="pluginsTab" >
      <attribute name="title" >
       <string>Plugins</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6" >
       <item>
        <layout class="QGridLayout" name="gridLayout_5" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_15" >
           <property name="text" >
            <string>Yaw axis</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="margin" >
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="yawAxisComboBox" >
           <property name="currentIndex" >
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_16" >
           <property name="text" >
            <string>Pitch Axis</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="margin" >
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2" >
          <widget class="QPushButton" name="pitchAxisConfigurePushButton" >
           <property name="text" >
            <string>Configure...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_17" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="text" >
            <string>Shutter</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="margin" >
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <widget class="QPushButton" name="yawAxisConfigurePushButton" >
           <property name="text" >
            <string>Configure...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" >
          <widget class="QPushButton" name="shutterConfigurePushButton" >
           <property name="text" >
            <string>Configure...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QComboBox" name="pitchAxisComboBox" />
         </item>
         <item row="2" column="1" >
          <widget class="QComboBox" name="shutterComboBox" />
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="driversTab" >
      <attribute name="title" >
       <string>Drivers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <item>
        <layout class="QGridLayout" name="gridLayout_9" >
         <item row="0" column="0" >
          <widget class="QLabel" name="bluetoothDeviceAddressLabel" >
           <property name="text" >
            <string>BT device address</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="margin" >
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="bluetoothDeviceAddressLineEdit" >
           <property name="text" >
            <string>00:50:C2:58:55:B9</string>
           </property>
           <property name="maxLength" >
            <number>50</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <widget class="QPushButton" name="bluetoothChoosePushButton" >
           <property name="text" >
            <string>Choose...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="serialPortLabel" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="text" >
            <string>Serial port</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="margin" >
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="serialPortLineEdit" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="text" >
            <string>0</string>
           </property>
           <property name="maxLength" >
            <number>50</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="ethernetHostPortLabel" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="text" >
            <string>Ethernet host/port</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="margin" >
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="ethernetHostLineEdit" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="text" >
            <string>locahost</string>
           </property>
           <property name="maxLength" >
            <number>50</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2" >
          <widget class="QSpinBox" name="ethernetPortSpinBox" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="minimum" >
            <number>1024</number>
           </property>
           <property name="maximum" >
            <number>65535</number>
           </property>
           <property name="value" >
            <number>7165</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QCheckBox" name="hardwareAutoConnectCheckBox" >
           <property name="text" >
            <string>Auto-connect</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>preferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>preferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
